architecture/2025-10-terraform-bicep-settings.md

```diff
@@ -0,0 +1,325 @@
+# Terraform & Bicep Settings Lifecycle Refresh
+
+- **Author**: Yetkin Timocin (@ytimocin)
+
+## Overview
+
+We will externalize Radius Terraform and Bicep recipe configuration into dedicated settings resources, centralize Terraform binary lifecycle, and let platform teams supply Terraform settings exactly as they do today. This keeps Radius orchestration intact while removing opinionated guardrails that block mature Terraform estates. The work is anchored to the feature spec [`2025-08-14-terraform-bicep-settings.md`](../features/2025-08-14-terraform-bicep-settings.md).
```

Comment:
Is this setting the standard for future, pluggable recipe engines? Is there a consideration for how we might make recipe engines pluggable instead of hard-coded?